TUM Campus App/MVGManager.swift

```diff
@@ -23,16 +29,36 @@ final class MVGManager: Manager, SimpleSingleManager {
         self.config = config
     }
     
+    func performRequest(maxCache: CacheTime) -> Promise<[Station], APIError> {
+        return config.mvg.doObjectsRequest(
```

Comment:
doObjectsRequest() is deprecated. Is there any other method to accomplish the desired functionality?

Reply:
Thats an issue with @nerdsupremacist sweeft library and should be addressed later.

Reply:
You’re right. For some reason, it looked to me like it was our own method